cmd/kueuectl/app/list/list_clusterqueue.go

```diff
@@ -117,7 +120,8 @@ func (o *ClusterQueueOptions) Complete(clientGetter util.ClientGetter, cmd *cobr
 func (o *ClusterQueueOptions) ToPrinter(headers bool) (printers.ResourcePrinterFunc, error) {
 	if !o.PrintFlags.OutputFlagSpecified() {
 		printer := newClusterQueueTablePrinter().
-			WithHeaders(headers)
+			WithHeaders(headers).
+			WithClock(o.Clock)
```

Comment:
Do you have default implementation of clock inside printer? What happens if we will not set `WithClock(o.Clock)`?

Reply:
You are right, I am currently working on this.
I think, that's the reason some tests fail.